README.md

```diff
@@ -2,23 +2,46 @@
 
 Library for building a Run on Slack Deno project. The artifacts produced from this library are what can be deployed as a Run on Slack project.
 
-The top level `mod.ts` file can be run as a Deno program. It takes up to 3 command line arguments:
+## Quickstart
 
-* `--manifest` If passed, will only generate the manifest and skip building functions
+In a directory that contains a valid manifest file (`manifest.json` or `manifest.ts`), run the following:
 
-* `--source` Absolute or relative path to your project. Defaults to current working directory.
+```
+deno run --unstable --allow-write --allow-read "https://deno.land/x/deno_slack_builder@0.0.5/mod.ts"
```

Comment:
nit: This is a static version — might be good to add a callout to use our latest version and point to our [releases page](https://github.com/slackapi/deno-slack-builder/releases)?

We've talked about a `latest` version but probably not ready to move on that.

Reply:
Thanks @shapirone! I added some text callout, and I'll keep my eye on new release versions to update this until a `latest` is available. My goal is to ensure that all code snippets will copy and run.